bittorrent/torrentfs.go

```diff
@@ -1,13 +1,14 @@
 package bittorrent
 
 import (
-	"os"
-	"time"
-	"sync"
+	"encoding/hex"
 	"errors"
-	"unsafe"
 	"net/http"
+	"os"
 	"path/filepath"
+	"sync"
+	"time"
+	"unsafe"
```

Comment:
Import order was fine here, `encoding/hex` would go just before `path/filepath` and that's it